ozaria/site/components/cutscene/common/BaseVideo.vue

```diff
@@ -9,39 +9,66 @@ export default {
       type: String,
```

Comment:
This file is also for turning cutscene video sound on and off. We are only implementing this for the Vimeo component as we're prioritizing the English flow first.